apps/meteor/client/views/omnichannel/directory/contacts/ContactTableRow.tsx

```diff
@@ -63,6 +64,9 @@ const ContactTableRow = ({ _id, name, phones, contactManager, lastChat, channels
 					<CallDialpadButton phoneNumber={phoneNumber} />
 				</GenericTableCell>
 			)}
+			<GenericTableCell withTruncatedText>
```

Comment:
I don't think you need `withTruncatedText` here
```suggestion
			<GenericTableCell>
```